### ISSUE TITLE
Decrease of modulesd read_ops in footprint test 

### ISSUE THREAD

**Issue by pro-akim** [wazuh/wazuh#18687]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 4.5.2-rc1 | Wazuh Manager | Manager | Packages | CentOS |

Jenkins build: https://ci.wazuh.info/job/Test_stress/4281/
Test issue: [Release 4.5.2 - Revision 1 - Release Candidate RC1 - Footprint Metrics - ALL-EXCEPT-SCA (2.5d)](https://github.com/wazuh/wazuh/issues/18671)

Big changes in read_ops of modulesd, decrease from 0-7.5 (4.5.1) to 0-1.75 (4.5.2)

4.5.2 rc1

![image](https://github.com/wazuh/wazuh/assets/125690423/e8f62a1e-7007-4346-a16f-96f34f3e7d7f)

4.5.1 rc2
![image](https://github.com/wazuh/wazuh/assets/125690423/ab2dde77-64f4-47f6-bf37-b75fce317f12)

It does not seem an issue, but further research should be performed

**Comment by LucioDonda** [comment]
Based on a not-so-deep analysis **4.5.2 rc1** modulesd read_ops (between **[0.75; 1.6E9]** )
are very similar as the one taken from 
**4.3.0 RC4** -> https://github.com/wazuh/wazuh/issues/12286 between **[1 ; 2E9]** and
**4.2.5 RC1** -> https://github.com/wazuh/wazuh/issues/12570 **[ 1 ; 2.25E9]**
All of them are similar. 
Nevertheless, **4.5.1** does have a significantly bigger slope **[1.7; 5.5E9]** which can create some confusion during analysis.
I'll try to dig deeper into the causes of this change, but from my side, it doesn't look like a metric to be worried about.

**Comment by LucioDonda** [comment]
As stated above the graph was compared with other footprint tests:
[4.4.2 - footprint all except sca (2.5d)](https://github.com/wazuh/wazuh/issues/17094)
![read_ops_442-1](https://github.com/wazuh/wazuh/assets/95248059/32521eac-1de8-46a6-92ce-d83967e1bb89)
And, as the last comment, the **read_ops** are between 1 and 2E9 during the full load.
Resulting in 4.5.1 being the only one outside those boundaries. 
To summarize:
* This behavior is very similar to previous versions and it doesn't look like something to worry about.
* Graph in 4.5.1 should have triggered some kind of alert in the previous footprint.
* Based on what was done in that version I couldn't find any possible cause for it. 
c.c.: @cborla , @pro-akim

**Comment by TomasTurina** [comment]
LGTM!
